docs/Admin/Test-ExchangePropertyPermissions.md

```diff
@@ -0,0 +1,30 @@
+# Test-ExchangePropertyPermissions
+
+Download the latest release: [Update-Engines.ps1](https://github.com/microsoft/CSS-Exchange/releases/latest/download/Update-Engines.ps1)
+
+## Description
+
+Test-ExchangePropertyPermissions is designed to detect certain schema issues which can manifest as
+permissions problems and can be challenging to identify manually, including:
+
+* Scenarios where a property set does not include all the expected properties.
+* Scenarios where an objectClass definition is missing expected properties.
+
+Note that the script does not perform an exhaustive check for all possible schema issues. It is
```

Comment:
I don't know how we would determine that. The script just tells you if the schema is borked. Whether it became borked via a known vector is up to the user to determine. I linked the AD Schema Analyzer article in the doc, because that's the common cause (all but 1 case so far).

We had one customer where there was somehow an attribute conflict that caused an attribute to be marked as a duplicate, but the original was gone, and the duplicate was incomplete. The script will identify that scenario for specific attributes. That's not a "known issue", but it's an issue we've seen.

For anything the script can't identify, it will most likely end up getting escalated to us, and we can add it to the script.

Reply:
okay